<ui version="4.0" >
 <class>TransferHistory</class>
 <widget class="QWidget" name="TransferHistory" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>278</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="clearButton" >
         <property name="text" >
          <string>Clear History</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KButtonGroup" name="viewmodesgroup" >
         <property name="title" >
          <string comment="View modes" >View Modes</string>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <property name="current" stdset="0" >
          <number>-1</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <property name="spacing" >
           <number>2</number>
          </property>
          <property name="sizeConstraint" >
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="margin" >
           <number>2</number>
          </property>
          <item>
           <widget class="KPushButton" name="bt_iconview" >
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
            <property name="autoExclusive" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="bt_listview" >
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoExclusive" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="KButtonGroup" name="kbuttongroup_2" >
         <property name="title" >
          <string>Select ranges</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <property name="spacing" >
           <number>2</number>
          </property>
          <property name="sizeConstraint" >
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="margin" >
           <number>2</number>
          </property>
          <item>
           <widget class="KComboBox" name="rangeType" >
            <item>
             <property name="text" >
              <string>Date</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Size</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Host</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="KButtonGroup" name="kbuttongroup_3" >
         <property name="title" >
          <string>Filter history</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <property name="spacing" >
           <number>2</number>
          </property>
          <property name="sizeConstraint" >
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="margin" >
           <number>2</number>
          </property>
          <item>
           <widget class="KLineEdit" name="searchBar" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDelete_Selected" >
   <property name="text" >
    <string comment="delete selected transfer" >Delete Selected</string>
   </property>
  </action>
  <action name="actionDownload" >
   <property name="text" >
    <string>Download</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
